typespec/Applications.Core/applications.tsp

```diff
@@ -39,23 +39,23 @@ namespace Applications.Core;
 
 @doc("Radius Application resource")
 model ApplicationResource is TrackedResource<ApplicationProperties> {
-  @doc("The application name")
+  @doc("The name of your resource.")
```

Comment:
For the resource descriptions let's try to avoid 2nd person terms like "your". Let's also include information about the resource type in the description as well.

Instead let's instead say something like:

```suggestion
  @doc("The name of the application. Used to identify and refer to the application and to form the full resource identifier.")
```